data_hub_api/docmaps/provider.py

```diff
@@ -26,21 +26,53 @@ def get_docmap_inputs_value_from_query_result(query_result_item: dict) -> list:
     }]
 
 
+def iter_single_actions_value_from_query_result_for_evaluations(
+    query_result_item: dict
+) -> Iterable[dict]:
+    evaluations = query_result_item['evaluations']
+    preprint_doi = query_result_item["preprint_doi"]
+    elife_doi = query_result_item['elife_doi']
+    url = f'https://doi.org/{elife_doi}'
+    for evaluation in evaluations:
+        hypothesis_id = evaluation["hypothesis_id"]
+        yield {
+            'participants': [],
+            'outputs': [
+                {
+                    'type': '',
+                    'doi': elife_doi,
+                    'published': evaluation['annotation_created_timestamp'] if evaluations else '',
```

Comment:
why do we have `if evaluations`? Wouldn't we then not get here because of the loop?

Reply:
Left from the previous code probably will remove it

Reply:
ups I have removed it already :)